src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -247,11 +251,11 @@ def generate_MCP_1_default_config(self, filename):
     # Create the actual config.
     iteration0_config = {
         'cbsdRequestsWithDomainProxies': [cbsd_records_iteration_0_domain_proxy_0,cbsd_records_iteration_0_domain_proxy_1],
-        'cbsdRecord' : {
-            'registrationRequest': device_7,
-            'grantRequest': grant_request_7,
-            'clientCert': os.path.join('certs', 'client.cert'), 
-            'clientKey': os.path.join('certs', 'client.key')
+        'cbsdRecords' : {
```

Comment:
Doesn't it make more sense to have a list of dictionaries instead of a dictionary of lists? You are much less likely to run into problems that way.

Reply:
we will study this and get back to you soon.

Reply:
Modified to list of  dictionaries to reduce ambiguity